packages/gatsby-theme-sso-ui-kit/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Partially out of scope of this PR as it would affect all the packages in the monorepo. Created separated issue https://github.com/commercetools/commercetools-docs-kit/issues/1517